components/filecoin/rs/Cargo.toml

```diff
@@ -7,16 +7,18 @@ publish = false
 
 [dependencies]
 base64 = "0.13.0"
-blake2b_simd = "1.0"
 bls-signatures = { version = "0.12", default-features = false, features = ["pairing"] }
 cxx = "1.0"
-fvm_shared = { version = "0.7.0 " }
+fvm_shared = { git = "https://github.com/brave-experiments/ref-fvm", branch = "3.2.0_downgraded_deps", default-features = false }
```

Comment:
If we're adding a dependency on this module, it needs to be moved to https://github.com/brave/ so that it's in scope for our bug bounty program. You can review [this checklist](https://github.com/brave/internal/wiki/GitHub-Security-Checklist) and then ask it-support to move it.

cc @kdenhartog who may want to take a look at it too

Reply:
All of the dependencies that are used in this branch have been audited previously in https://github.com/brave/security/issues/840 and are approved in https://github.com/brave/handbook/blob/master/development/crypto-dependencies.md.

As long as we follow the guidance from @fmarier above this approach LGTM